test/linalg/tridiag.jl

```diff
@@ -50,9 +50,21 @@ for elty in (Float32, Float64, Complex64, Complex128, Int)
     @test ctranspose(T) == Tridiagonal(conj(du), conj(d), conj(dl))
 
     @test abs.(T) == Tridiagonal(abs.(dl),abs.(d),abs.(du))
+    if elty <: Real
+        @test round.(T) == Tridiagonal(round.(dl),round.(d),round.(du))
```

Comment:
I think `isa` because the most likely problem is that these suddenly end up as `Array`s.